### ISSUE TITLE
Synth item emits error in prepare phase

### ISSUE THREAD

**Issue by dschreij** [shyras/osweb#167]
When running an experiment with a synth, it raises an error during the prepare phase. The message of the error is also not clear, because OSWeb error handling function does not work properly either (e.g. you get an 'addError' not found' message).

**Comment by dschreij** [comment]
I notice that the implementation of synth is all but complete! It simply loads a sampler item without a file, causing this error. @smathot do you agree we should add this item to the unsupported items list (and/or implement it for 1.4)?

**Comment by smathot** [comment]
Yes, I agree. There's no unsupported-items list ;-) But I will remove it from the supported-items list.
